pysteps/utils/arrays.py

```diff
@@ -11,6 +11,81 @@
 """
 
 import numpy as np
+from numpy.lib.stride_tricks import as_strided
+
+
```

Comment:
This new method seems very similar to the already existing `utils.dimensions.aggregate_fields_space` method, although with improved functionalities (e.g., data are trimmed if not perfectly divisible).

Do we need to keep both? Can we have only one method to upscale/reduce/aggregate fields in space?